pkg/skaffold/deploy/helm.go

```diff
@@ -165,7 +180,62 @@ func (h *HelmDeployer) deployRelease(out io.Writer, r v1alpha2.HelmRelease, buil
 	if len(r.Overrides) != 0 {
 		os.Remove("skaffold-overrides.yaml")
 	}
-	return helmErr
+
+	return h.getDeployResults(ns, r.Name), helmErr
+}
+
+func (h *HelmDeployer) getReleaseInfo(release string) (*bufio.Reader, error) {
+	var releaseInfo bytes.Buffer
+	out := bufio.NewWriter(&releaseInfo)
```

Comment:
You shouldn't need this bufio.NewWriter, you can use a `var releaseInfo bytes.Buffer` directly as a reader/writer:
https://golang.org/pkg/bytes/#Buffer.Write

Reply:
yep, thanks